themes/edx.org/lms/templates/header.html

```diff
@@ -104,6 +104,10 @@ <h2 class="course-header">
 
       <%include file="user_dropdown.html"/>
 
+      <a href="${get_online_help_info(online_help_token)['doc_url']}" 
```

Comment:
I'd wrap this with a div or maybe a ul tag, so that it becomes a more formal part of the header. I don't think that the help link is special in that sense, and so I could see us adding other links here in the future.